packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -37,29 +61,27 @@ class InteractionLayerContainer extends Component {
   }
 
   render () {
-    const { activeStepTasks } = this.props
-    const isDrawingInActiveSteps = activeStepTasks && activeStepTasks.some(task => task.type === 'drawing')
-
     return (
       <>
         <InteractionLayer
           onPointerDown={this.onPointerDown}
           onPointerMove={this.onPointerMove}
           onPointerUp={this.onPointerUp}
         />
-        {isDrawingInActiveSteps && <DrawingContainer />}
+        {this.props.isDrawingInActiveWorkflowStep && <DrawingContainer />}
```

Comment:
If the next step in the workflow isn't a drawing task, all my marks from this step will vanish when I press Next.

Reply:
I think the conditional here should be on the drawing canvas AKA interaction layer. Any marks that have been drawn should always be rendered, but we don't always want to enable drawing.

`InteractionLayer` and `DrawingContainer` should be renamed to describe what they actually do.